framework/src/base/FEProblemBase.C

```diff
@@ -672,6 +672,14 @@ void FEProblemBase::initialSetup()
     for (THREAD_ID tid = 0; tid < n_threads; ++tid)
         _assembly[tid]->initNonlocalCoupling();
   }
+
+  // Remove some of the heavier objects that are only needed during initial
+  // Several other types of objects may be added here in the future.
+  _multi_apps.clear(EXEC_INITIAL);
```

Comment:
No - I put that in the documentation and provided a test case to show that it works properly.